final-project/scripts/blog.js

```diff
@@ -0,0 +1,96 @@
+const url = "https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog";
+
+async function getBlogItems() {
+  try {
+    return new Promise(resolve => fetch(url).then(response => resolve(response.json())));
+  } catch (err) {
+    throw err;
+  }
+}
+
+function createBlogItem(data, index) {
+  const  itemId = index;
+  const { images, title, description, url } = data;
+  let image;
+  if (screen.width > 768) image = images.desktop;
+  else if (screen.width > 400) image = images.tablet;
+  else image = images.mobile;
+  const template = `
+  <div class="blog__carrousel__inner__item" id="blog-item-${itemId}">
```

Comment:
Just as a recommendation, this could have been hidden element in your HTML and get cloned by js, just in case the structure changes, it would be easier to handle there. For this exercise this is ok.